Spigot-Server-Patches/0307-Vanished-players-don-t-have-rights.patch

```diff
@@ -0,0 +1,131 @@
+From f53716297108475886984bbfda58513ce80077d7 Mon Sep 17 00:00:00 2001
+From: Hugo Manrique <hugmanrique@gmail.com>
+Date: Mon, 23 Jul 2018 14:22:26 +0200
+Subject: [PATCH] Vanished players don't have rights
+
+
+diff --git a/src/main/java/net/minecraft/server/Entity.java b/src/main/java/net/minecraft/server/Entity.java
+index b6d6d4f3..ce2fdbeb 100644
+--- a/src/main/java/net/minecraft/server/Entity.java
++++ b/src/main/java/net/minecraft/server/Entity.java
+@@ -92,7 +92,7 @@ public abstract class Entity implements INamableTileEntity, ICommandListener, Ke
+     private static int entityCount;
+     private final EntityTypes<?> g; public EntityTypes<?> getEntityType() { return g; } // Paper - OBFHELPER
+     private int id;
+-    public boolean j;
++    public boolean j; public boolean blocksEntitySpawning() { return j; } // Paper - OBFHELPER
+     public final List<Entity> passengers;
+     protected int k;
+     private Entity ax;
+diff --git a/src/main/java/net/minecraft/server/IBlockData.java b/src/main/java/net/minecraft/server/IBlockData.java
+index c8f305e6..d3103cb4 100644
+--- a/src/main/java/net/minecraft/server/IBlockData.java
++++ b/src/main/java/net/minecraft/server/IBlockData.java
+@@ -183,6 +183,8 @@ public interface IBlockData extends IBlockDataHolder<IBlockData> {
+         return this.getBlock().f(this, iblockaccess, blockposition);
+     }
+ 
++    default VoxelShape getBlockShape(IBlockAccess iblockaccess, BlockPosition blockposition) { return h(iblockaccess, blockposition); } // Paper - OBFHELPER
```

Comment:
Please put this on the line *directly* (no blank line between) above the method the obfuscation helper is for.
Example:
```java
default VoxelShape getBlockShape(IBlockAccess iblockaccess, BlockPosition blockposition) { return h(iblockaccess, blockposition); } // Paper - OBFHELPER
// <h(IBlockAccess, BlockPosition) method on this line>
```